packages/form-js-editor/src/features/properties-panel/entries/DateTimeConstraintsEntry.js

```diff
@@ -0,0 +1,104 @@
+import { CheckboxEntry, isCheckboxEntryEdited, SelectEntry, isSelectEntryEdited } from '@bpmn-io/properties-panel';
+
+import { DATETIME_SUBTYPES, DATETIME_DISALLOWPASTDATES_PATH, TIME_INTERVAL_PATH } from '@bpmn-io/form-js-viewer';
+
+import { get } from 'min-dash';
+
+export default function DateTimeConstraintsEntry(props) {
+  const {
+    editField,
+    field,
+    id
+  } = props;
+
+  const {
+    type,
+    subtype
+  } = field;
+
+  if (type !== 'datetime') {
+    return [];
+  }
+
+  const entries = [];
+
+  if (subtype === DATETIME_SUBTYPES.TIME || subtype === DATETIME_SUBTYPES.DATETIME) {
+    entries.push({
+      id: id + '-timeInterval',
+      component: TimeIntervalSelect,
+      isEdited: isSelectEntryEdited,
+      editField,
+      field
+    });
+  }
+
+  if (subtype === DATETIME_SUBTYPES.DATE || subtype === DATETIME_SUBTYPES.DATETIME) {
+    entries.push({
+      id: id + '-disallowPassedDates',
+      component: DisallowPassedDates,
+      isEdited: isCheckboxEntryEdited,
+      editField,
+      field
+    });
+  }
+
+  return entries;
+}
+
+function DisallowPassedDates(props) {
+  const {
+    editField,
+    field,
+    id
+  } = props;
+
+  const path = DATETIME_DISALLOWPASTDATES_PATH;
+
+  const getValue = () => {
+    return get(field, path, '');
+  };
+
+  const setValue = (value) => {
+    return editField(field, path, value);
+  };
+
+  return CheckboxEntry({
+    element: field,
+    getValue,
+    id,
+    label: 'Disallow past dates',
+    setValue
+  });
+}
+
+function TimeIntervalSelect(props) {
+
+  const {
+    editField,
+    field,
+    id
+  } = props;
+
+  const timeIntervals = [ 1, 5, 10, 15, 30, 60 ];
```

Comment:
Ended up not doing that because of the bespoke logic turning '60' into '1h' in the dropdown. I could make this behavior more generic though and always visually convert it to hh:mm, then it would make sense to extract. Not sure if it's worth the effort though. LMK what you think.